scripts/microbe_stage/microbe_editor/microbe_editor.as

```diff
@@ -945,7 +962,7 @@ class MicrobeEditor{
 
         } else if(type == "MicrobeEditorExited"){
             LOG_INFO("MicrobeEditor: applying changes to player Species");
-
+            getNucleus();
```

Comment:
I'm still not convinced that this is bullet proof. The check should happen when entering the editor. And placing a nucleus should also update it.

And to avoid confusion the buttons for organelles that can't be placed should be disabled.

Reply:
So the check should be made when we enter in Editor, and enabling or not some GUI buttons.
And once you have put a nucleus this functions shouldn't be call again in next edits.

Do you agree?

Reply:
The function should be called: when entering, when placing a nucleus, when redoing a nucleus action, and when undoing a nucleus action. It might be cleaner to run it when entering and then check if something changed when each action is applied or undone.

Reply:
Ah ok, but if nucleus cost 100 points if i run it when i enter in editor and  when i go out should be enough i think.

Reply:
I'm considering all possible scenarios, one of which is that you are playing around with a freebuild editor, where you have unlimited MP.

Reply:
Ok, in that case it won't work, the problem is that in function undo e redu scope is different, i dont't think class function can be run.